src/components/publishing/__stories__/publishing.tsx

```diff
@@ -8,15 +8,37 @@ import Image from "../image"
 import ImageCollection from "../image_collection"
 import ImagesetPreview from "../imageset_preview"
 
-import { FeatureHeaders, Images } from "../__test__/fixtures"
+import { Artworks, FeatureHeaders, Images } from "../__test__/fixtures"
 
 import Typography from "./typography"
 
 storiesOf("Publishing", Artwork)
   .add("Artwork", () => {
     return (
-      <div style={{ width: 400 }}>
-        <Artwork linked artwork={Images[0]} />
+      <div>
+        <div style={{ width: 800 }}>
+          <Artwork linked artwork={Artworks[0]} />
+        </div>
+        <hr />
+        <p>Multiple Artists: </p>
+        <div style={{ width: 800 }}>
+          <Artwork linked artwork={Artworks[1]} />
+        </div>
+        <hr />
+        <p>Unlinked: </p>
+        <div style={{ width: 800 }}>
+          <Artwork linked={false} artwork={Artworks[1]} />
+        </div>
```

Comment:
nope! but that brings up a good point which is that our default should probably be `true` for `linked`.